tasks/task.json

```diff
@@ -44,6 +44,18 @@
           }
         }
       }
+    },
+    "files": {
```

Comment:
Need to also update the `implementations` object properties.

Reply:
Not exactly sure how to express the expected "items" `implementations` expects. Tried to express as an array of the two possible hash objects.
 {"name": "foo.ps1", "requirements": ["powershell"]}
 {"files": "my_util/files/util/win_tools.ps1", "requirements": ["powershell"]}